### PR TITLE
update commons-io to latest (2.6)

### PR DESCRIPTION
**Pull request by tysonnorris**
update to latest commons-io version

### CODE REVIEW

File: tests/src/test/scala/common/rest/WskRest.scala [hunk 1]
```diff
@@ -278,16 +270,25 @@ class WskRestAction
             ("", Base64.getEncoder.encodeToString(zip), artifactFile)
           }
           case ".js" => {
-            ("nodejs:default", FileUtils.readFileToString(new File(artifactFile)), artifactFile)
+            (
+              "nodejs:default",
+              FileUtils.readFileToString(new File(artifactFile), Charset.forName("UTF-8")),
```
Comment (markusthoemmes) [inline tests/src/test/scala/common/rest/WskRest.scala]: Replacing all of the charset getters with `StandardCharsets.UTF_8` would be nice.

**Comment by csantanapr** [comment]
PG started Playground3/1651

**Comment by tysonnorris** [comment]
Merging this PR must be done BEFORE https://github.com/apache/incubator-openwhisk-cli/pull/195

And https://github.com/apache/incubator-openwhisk-cli/pull/195 must be merged immediately after merge of this PR.

**Comment by csantanapr** [comment]
PG2 2672 started

**Comment by csantanapr** [comment]
🔵  on PG2 2673

**Pull request merged**